packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -833,15 +768,19 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
               </div>
             </section>
 
-            {/* Queue location and queue fields. These get shown when queue location and queue fields are configured */}
-            {config.showServiceQueueFields && (
-              <section>
-                <div className={styles.sectionTitle}></div>
-                <div className={styles.sectionField}>
-                  <ExtensionSlot name="visit-form-queue-slot" state={{ setFormFields: setVisitFormFields }} />
-                </div>
-              </section>
-            )}
+            {/* Queue location and queue fields. These get shown when config.showServiceQueueFields is true,
+                or when the form is opened from the queues app */}
+            <section>
+              <div className={styles.sectionTitle}></div>
+              <div className={styles.sectionField}>
+                <VisitFormExtensionSlot
+                  name="visit-form-queue-slot"
+                  patientUuid={patientUuid}
+                  visitFormOpenedFrom={openedFrom}
+                  setOnVisitCreatedOrUpdatedCallbacks={setOnVisitCreatedOrUpdatedCallbacks}
```

Comment:
More a question than a request for a change... why do the queues and appointments have specific extension slots, rather than a single "visit-form-extension-slot" that both queues and appointments insert into?  As we have it now, if another ESM wanted to insert it's own functionality into the visit form, would it have to add another extension slot to the form?

Reply:
Yeah, with this change, those 2 extensions look more similar than before. I'm keeping it as is because where the extensions appear are different (the appointments extension is near the top, close to where the visit start time fields are, vs the queues extension is near the bottom). It might be possible to slot them into one single slot (and keeping the existing ordering) if we slice up the visit form further so that even the visit form fields are extensions themselves , but I think that's a pretty drastic change.

Reply:
I don't _love_ that the slots are named after specific apps ... It kind of violates the idea that slots are just a location on a screen that extensions can appear. I'm kind of ok with having multiple slots if it makes sense, e.g., there are two places on a screen we want extensions to appear.

Reply:
Ok, I changed the slot named to `visit-form-top-slot` and `visit-form-bottom-slot`

Reply:
Thanks @chibongho  and @ibacher !